lib/tasks/after_tasks.rake

```diff
@@ -788,15 +788,57 @@ namespace :After do
       STDOUT.flush
     end
 
-    puts "Converted #{borked_rating_tag.name} rating tag on #{works_using_tag.size - invalid_works.size} works"
+    puts "Converted '#{borked_rating_tag.name}' rating tag on #{works_using_tag.size - invalid_works.size} works"
     STDOUT.flush
   end
 
+  desc "Clean up noncanonical rating tags"
+  task(clean_up_noncanonical_ratings: :environment) do
+    canonical_not_rated_tag = Rating.find_by!(name: ArchiveConfig.RATING_DEFAULT_TAG_NAME)
+    noncanonical_ratings = Rating.where(canonical: false)
+    puts "There are #{noncanonical_ratings.size} noncanonical rating tags."
+
+    unless noncanonical_ratings.empty?
```

Comment:
If `noncanonical_ratings.empty?`, do we even need to go through the rest of the task?

Reply:
We don't, and the rest is no-op. I've added code to [return early from the task](https://stackoverflow.com/questions/2316475/how-do-i-return-early-from-a-rake-task).